new/LIT7288PrBaryaDis.xml

```diff
@@ -0,0 +1,78 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7288PrBaryaDis" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1"> ጸሎት፡ በእንተ፡ ዓይነተ፡ ባርያ፡ መጋኛ፡ ወሾቶላይ፡ ውግዓት፡ ወቍርጥማት፡ ፍልፀት፡</title>
+                <title xml:lang="gez" type="normalized" corresp="#t1">Ṣalot baʾǝnta ʿāynata Bāryā Magāññā wa-Šotolāy wǝgʿāt wa-qʷǝrṭǝmāt
+                    fǝlḍat</title>
+                <title xml:lang="en" corresp="#t1">Prayer against the evil eye of Bāryā, Magāññā, and Šotolāy, chest pain, rheumatism, pleurisy, ...</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>Strelcyn reported in <bibl><ptr target="bm:Strelcyn1978BritishLibrary"/><citedRange unit="page">136</citedRange></bibl>,
```

Comment:
Maybe this could be syntactically clarified a little? For example:

```
Strelcyn reported in <bibl><ptr target="bm:Strelcyn1978BritishLibrary"/><citedRange unit="page">136</citedRange></bibl> 
that this prayer is against blacksmiths, among other things. 
Blacksmiths are usually referred to as <foreign xml:lang="gez">ነሃቢ፡</foreign>, while this
                    text instead uses the term <foreign xml:lang="gez">ጠቢብ፡</foreign>, which can also have many other meanings, such as "wise man / scholar / physician / 
                    artisan / expert / craftsman".
```